src/plugins/kibana_usage_collection/server/collectors/saved_objects_counts/kibana_usage_collector.ts

```diff
@@ -29,7 +29,7 @@ export async function getKibanaSavedObjectCounts(
   esClient: ElasticsearchClient,
   kibanaIndex: string
 ): Promise<KibanaSavedObjectCounts> {
-  const buckets = await getSavedObjectsCounts(esClient, kibanaIndex, TYPES);
+  const { per_type: buckets } = await getSavedObjectsCounts(esClient, kibanaIndex, TYPES, true);
```

Comment:
We can probably deprecate/remove this collector now and rather just depend on one collector for saved objects counts? Maybe we should create an issue to ask any teams that might rely on this collector to switch to the new one?

Reply:
There is an issue already for it: #99464

It's pending confirmation from the telemetry team to learn if any existing dashboard uses it.